src/lib/RouteConfig.js

```diff
@@ -66,14 +55,12 @@ export default (app: $Application, controllers: Array<Controller>) => {
         return;
       }
 
-
-      const argumentNames = getFunctionArgumentNames(mappedFunction);
-
       app[httpVerb](
         route,
         maybe(oauth.authenticate(), !anonymous),
         injectUserMiddleware(),
         async (request: $Request, response: $Response) => {
+          const argumentNames = request.route.path.split('/:').splice(1);
```

Comment:
since `:[\w]*/g` is not good enough and its 3:40 pm atm, I end up with this.. will think about the right regexp next time :)